testsuite/MDAnalysisTests/topology/test_gro.py

```diff
@@ -90,38 +84,36 @@ class TestGroResidWrapping(object):
     lengths = [19, 24, 19, 19, 19, 19, 7]
     parser = mda.topology.GROParser.GROParser
 
+    @pytest.mark.parametrize('parser, resids', (
+        (GRO_residwrap, [1, 99999, 100000, 100001, 199999, 200000, 200001]),
+        (GRO_residwrap_0base, [0, 99999, 100000, 100001, 199999, 200000,
+                               200001])
 
-    def test_wrapping_resids(self):
-        parser = mda.topology.GROParser.GROParser
-        with self.parser(GRO_residwrap) as p:
+    ))
+    def test_wrapping_resids(self, parser, resids):
+        with self.parser(parser) as p:
             top = p.parse()
 
-        resids = [1, 99999, 100000, 100001, 199999, 200000, 200001]
-
-        assert_(top.tt.size == (126, 7, 1))
+        assert top.tt.size == (126, 7, 1)
         for i, (r, n, l) in enumerate(zip(resids, self.names, self.lengths)):
```

Comment:
We probably can compare the full arrays rather than comparing element per element.